bin/list-nginx-class-ingresses.rb

```diff
@@ -1,28 +1,27 @@
 #!/usr/bin/env ruby
 
-# Script to migrate multiple ingresses to a different ingress controller, by
-# deploying a second ingress on that controller, and updating the route53 TXT
-# record for each of the ingress's hostnames
+# Script to build the list of ingresses and its namespaces with 
+# the ingress-class annotations as "nginx" or empty i.e "" or ingress
+# with no annotations and into a json file 
 
-# TODO: remove this
-require "pry-byebug"
 require "json"
 require "open3"
 
+NGINX_CLASS_INGRESS_LIST_FILE = "nginx_class_ingresses.json"
+
 def main
-  nginx_class_ingresses = build_ingresses_has_annotations("nginx")
-  null_class_ingresses = build_ingresses_has_annotations("")
-  no_annotation_ingresses = build_ingresses_no_annotations
+  nginx_class_ingresses = list_ingresses_has_annotations("nginx")
+  null_class_ingresses = list_ingresses_has_annotations("")
+  no_annotation_ingresses = list_ingresses_no_annotations
 
   nginx_class_ingresses.concat(null_class_ingresses)
```

Comment:
Sorry @poornima-krishnasamy, I missed something here.

When you've got lists in ruby, and you want to join them together, you don't need to call `concat`, you can just use `+`

So, I'd probably use something like this:

```
list = nginx_class_ingresses + null_class_ingresses + no_annotation_ingresses
File.write(NGINX_CLASS_INGRESS_LIST_FILE, list.to_json)
```

Reply:
@digitalronin , thanks for the suggestion. I have updated the code for that.